src/frontend/src/tests/btc/components/send/BtcSendReview.spec.ts

```diff
@@ -0,0 +1,153 @@
+import BtcSendReview from '$btc/components/send/BtcSendReview.svelte';
+import * as btcPendingSendTransactionsStatusStore from '$btc/derived/btc-pending-sent-transactions-status.derived';
+import { BTC_MAINNET_TOKEN } from '$env/tokens/tokens.btc.env';
+import { REVIEW_FORM_SEND_BUTTON } from '$lib/constants/test-ids.constants';
+import { SEND_CONTEXT_KEY } from '$lib/stores/send.store';
+import { mockBtcAddress, mockUtxosFee } from '$tests/mocks/btc.mock';
+import en from '$tests/mocks/i18n.mock';
+import { mockPage } from '$tests/mocks/page.store.mock';
+import { render, waitFor } from '@testing-library/svelte';
+import { readable } from 'svelte/store';
+
+describe('BtcSendReview', () => {
+	const mockContext = (balance: bigint | undefined = 1000000n) =>
+		new Map([
+			[
+				SEND_CONTEXT_KEY,
+				{
+					sendToken: readable(BTC_MAINNET_TOKEN),
+					sendTokenDecimals: readable(BTC_MAINNET_TOKEN.decimals),
+					sendTokenId: readable(BTC_MAINNET_TOKEN.id),
+					sendTokenStandard: readable(BTC_MAINNET_TOKEN.standard),
+					sendTokenSymbol: readable(BTC_MAINNET_TOKEN.symbol),
+					sendBalance: readable(balance)
+				}
+			]
+		]);
+	const props = {
+		destination: mockBtcAddress,
+		source: mockBtcAddress,
+		amount: 0.0001,
+		networkId: BTC_MAINNET_TOKEN.network.id,
+		utxosFee: mockUtxosFee
+	};
+	const mockBtcPendingSendTransactionsStatusStore = (
+		status:
+			| btcPendingSendTransactionsStatusStore.BtcPendingSentTransactionsStatus
+			| undefined = btcPendingSendTransactionsStatusStore.BtcPendingSentTransactionsStatus.NONE
+	) =>
+		vi
+			.spyOn(btcPendingSendTransactionsStatusStore, 'initPendingSentTransactionsStatus')
+			.mockImplementation(() => readable(status));
+
+	const buttonTestId = REVIEW_FORM_SEND_BUTTON;
+
+	beforeEach(() => {
+		mockPage.reset();
+	});
+
+	it('should keep the next button clickable if all requirements are met', () => {
```

Comment:
nitpick: not sure why you use "should keep" in the titles. 

e.g. 

"should keep the next button clickable if all requirements are met"

vs

"should enable the next button if all requirements are met"

Reply:
Updated 👍